src/Fmt_ast.ml

```diff
@@ -1632,7 +1644,8 @@ and fmt_expression c ?(box= true) ?epi ?eol ?parens ?ext ({ast= exp} as xexp)
         ( Cmts.fmt c.cmts loc @@ fits_breaks_if parens "" "("
         $ fits_breaks "" (if override then "let open! " else "let open ")
         $ fmt_longident txt $ fits_breaks opn " in"
-        $ fmt_or_k force_fit_if (fmt "@;<0 2>")
+        $ fmt_or_k force_fit_if
+            (fits_breaks "" "@;<0 2>")
```

Comment:
I'm sorry, my suggestion to change this was a mistake, it does more harm than good.

I reverted it locally and tested, and all else looks good. So if you could revert this, and rebase to check the tests, it's ready to go.